rerun_py/rerun/__init__.py

```diff
@@ -10,17 +10,26 @@
 When we encounter this file on import, we instead redirect to the
 real rerun module by adding it to the path and then, and then
 replacing our own module content with it.
+
+Additionally, we set the RERUN_CLI_PATH environment variable to point
+to the rerun binary in the target directory so that we don't need to
+inject it into the source tree.
 """
 
 from __future__ import annotations
 
+import os
 import pathlib
 import sys
 
 real_path = pathlib.Path(__file__).parent.parent.joinpath("rerun_sdk").resolve()
 
 print(f"DEV ENVIRONMENT DETECTED! Re-importing rerun from: {real_path}", file=sys.stderr)
 
+if "RERUN_CLI_PATH" not in os.environ:
+    target_path = pathlib.Path(__file__).parent.parent.parent.joinpath("target/debug/rerun").resolve()
```

Comment:
Should we have a sanity check that this file exists?

Reply:
I added one to the place where do the the execution that now gives a cleaner error message.